pymongo/asynchronous/encryption.py

```diff
@@ -142,6 +142,37 @@ def __init__(
         self.opts = opts
         self._spawned = False
 
+    async def _async_kms_request(
```

Comment:
I'd like to avoid duplicating this method. What motivated this change?

Reply:
Good catch, I think this method could be collapsed into the existing `if _IS_SYNC` block inside `kms_request`.